computer_science_course_prereqs.csv

```diff
@@ -0,0 +1,41 @@
+Course,Prerequisite(s),,
```

Comment:
we only need two columns right?
Course and Prerequisite? right?

Reply:
unfortunately, no
lots of classes have multiple prereqs.

Reply:
we need to decide how we want to handle courses with multiple prereqs

Reply:
@Keggeraider it is easier for our code to just see the same course listed multiple times where each apeparance means it has one more prerequ.

example
```
CS3400,CS3300
CS3400,MATH1302
```

basically says CS3400 has two prereqs.